src/esp/gfx/PbrDrawable.cpp

```diff
@@ -56,13 +58,17 @@ void PbrDrawable::setMaterialValuesInternal(
   matCache.metalness = tmpMaterialData.metalness();
   matCache.emissiveColor = tmpMaterialData.emissiveColor();
 
-  if (materialData_->hasAttribute("metallicTexturePointer") &&
-      materialData_->hasAttribute("roughnessTexturePointer")) {
+  const Cr::Containers::Optional<Mn::GL::Texture2D*> metallicTexturePtr =
```

Comment:
This is pretty verbose. Could we use `auto` for the type of `metallicTexturePtr`? And/or a local, reusable lambda function?

Reply:
I don't like `auto` but I think it's fine to use it in this case -- the type is almost clear from the findAttribute call below.